sources/SoundcloudSDK/soundcloud/SouncloudClient.swift

```diff
@@ -354,14 +355,17 @@ public class SoundcloudClient: NSObject {
         protocolType: .HTTPS)
 
     /// The session property is only set when a user has logged in.
-    public fileprivate(set) static var session: Session? = {
-        if let data = keychain.data(forKey: sessionKey), let session = NSKeyedUnarchiver.unarchiveObject(with: data) as? Session {
-            return session
+    public fileprivate(set) static var session: Session? {
+        get {
```

Comment:
Hm actually I did it the way it was so that we don't access the keychain/unarchive the session every time we access the session.